tests/rados/test_bluestoretool_workflows.py

```diff
@@ -50,7 +50,7 @@ def run(ceph_cluster, **kw):
 
     try:
         # Execute ceph-bluestore-tool --help
-        osd_id = random.randrange(len(osd_list))
+        osd_id = random.choice(osd_list)
```

Comment:
Any reason for changing from randrange to choice?

Reply:
Yes, We need to select OSDs that are part of cluster, not just randomly select OSDs in a range. 

Failure log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-LVA8L4/ceph-bluestore-tool_utility_0.log